calico/felix/test/test_fetcd.py

```diff
@@ -45,6 +45,7 @@ class TestEtcdAPI(BaseTestCase):
     @patch("gevent.spawn", autospec=True)
     def test_create(self, m_spawn, m_etcd_watcher):
         m_config = Mock(spec=Config)
+        m_config.ETCD_ADDR = 'localhost:4001'
```

Comment:
Move 'localhost:4001' into a constant somewhere rather than repeatedly hard coding it. Makes life easier if etcd_addr ever changes in future!